webapp/src/ts/reducers/contacts.ts

```diff
@@ -1,15 +1,16 @@
-import { Actions } from '../actions/contacts';
 import { createReducer, on } from '@ngrx/store';
 import { UniqueSortedList } from './utils';
+
 import { ContactTypesService } from '@mm-services/contact-types.service';
+import { Actions } from '@mm-actions/contacts';
 
 const initialState = {
   contacts: [],
   contactsById: new Map(),
   selected: null,
   filters: {},
   loadingSelectedChildren: false,
-  loadingSelectedContacts: false,
+  loadingSelectedReports: false,
```

Comment:
`loadingSelectedContacts` isn't used anymore in the code.
Added `loadingSelectedReports` in state definition.